signal_scanner_bot/env.py

```diff
@@ -13,8 +14,43 @@
 class _State:
     """Class for holding global state across threads/tasks"""
 
-    LISTENING = False
-    STOP_REQUESTED = False
+    # Initialize state, set LISTENING to true if state file exists,
+    # sets false if not.
+    def __init__(self, file: str):
+        self.file = file
+        self.LISTENING = True if os.path.isfile(file) else False
+        self.STOP_REQUESTED = False
+
+    # Method to update the listening status of the State class
+    # object. Checks for on/off and creates/deletes state file.
+    def update_listening_status(self, status: str) -> Optional[str]:
+        if status == "AUTOSCANON":
```

Comment:
We'll want to move the `START_LISTENING` and `STOP_LISTENING` string constants from `messages.py` into this file, so we can reference those instead of the strings directly.

Reply:
Thought about moving this into the .env but decided not to. We can revisit later maybe but it feels unnecessary. Also moved the `==Auto Scanning Activated==` and `==Auto Scanning Deactivated==` notifications out to be constants in the env.py file.